packages/contentful/package.json

```diff
@@ -16,6 +16,7 @@
     "schema:update:crn": "graphql-codegen --config codegen.yml"
   },
   "dependencies": {
+    "contentful-migrate": "^0.17.0",
```

Comment:
this is fine i just wanted to point out that this is a duplicate of what you have in the root package.json, but as mentioned earlier it should be the only place you add it

Reply:
This is removed from root package.